.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Create release App Bundle
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+
+# Enrich gradle.properties for CI/CD
+env:
```

Comment:
It's missing maptiler's api key secret.

Reply:
I've added them in here: https://github.com/vector-im/element-x-android/pull/926